cmd/deploy.go

```diff
@@ -30,7 +30,7 @@ NAME
 
 SYNOPSIS
 	{{rootCmdUse}} deploy [-R|--remote] [-r|--registry] [-i|--image] [-n|--namespace]
-	             [-e|env] [-g|--git-url] [-t|git-branch] [-d|--git-dir]
+	             [-e|--env] [-g|--git-url] [-t|git-branch] [-d|--git-dir]
```

Comment:
```suggestion
	             [-e|--env] [-g|--git-url] [-t|--git-branch] [-d|--git-dir]
```
Nice catch - there's one more!